src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-defaultButtonWithIcon/Components/index.test.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { mount, render } from 'enzyme';
 
 import { AppIcon } from '../../../../types/iconsTypes/iconEnums';
-import DefaultButtonWithIcon from './defaultButtonWithIcon';
+import { DefaultButtonWithIcon } from '.';
```

Comment:
Let's be consistent.

```suggestion
import { DefaultButtonWithIcon } from './index';
```

Reply:
Made changes